main-app/app/less/main

```diff
@@ -78,14 +171,79 @@
       }
 
       td {
+//        button.image {
+//          //opacity: 1;
+//          //transition: 0.2s;
+//        }
+
         button {
-          width: 120px;
-          height: 120px;
+          //opacity: 0;
+//          width: @cell-length;
+//          height: @cell-length;
           font-size: 30px;
-          border-color: @light-blue-colour;
-          background-color: @light-blue-colour;
+//          border-color: @cell-colour;
+//          background-color: @cell-colour;
         }
       }
     }
   }
-}
\ No newline at end of file
+}
+
+
+.buildThemes(@index) when (@index < (@numberOfThemes + 1)) {
```

Comment:
This is impressive, but wouldn't it be simpler to have a base theme which covers all three themes (e.g. box-shadow), then override for each style with the required colour etc?